src/js/tabs/pid_tuning.js

```diff
@@ -977,21 +953,18 @@ TABS.pid_tuning.initialize = function (callback) {
                 const checked = $(this).is(':checked');
                 let cutoff = FC.FILTER_CONFIG.gyro_lowpass2_hz > 0 ? FC.FILTER_CONFIG.gyro_lowpass2_hz : FILTER_DEFAULT.gyro_lowpass2_hz;
 
-                if (TuningSliders.sliderGyroFilter) {
-                    cutoff = checked ? Math.floor(FILTER_DEFAULT.gyro_lowpass2_hz * TuningSliders.sliderGyroFilterMultiplier) : 0;
-                    FC.FILTER_CONFIG.gyro_lowpass2_hz = cutoff;
-                    TuningSliders.updateFiltersInFirmware();
+                if (FC.TUNING_SLIDERS.slider_gyro_filter) {
+                    cutoff = checked ? Math.floor(FILTER_DEFAULT.gyro_lowpass2_hz * FC.TUNING_SLIDERS.slider_gyro_filter_multiplier / 100) : 0;
```

Comment:
Hi Mark.
This code calculates gyro lowpass2 cutoff based on the slider value.
However, this code is inside 4.3 section. And for 4.3 the Configurator should not be calculating cutoffs like that. Instead, it should be taking the values from the firmware.

The current implementation here leads to the following misconception:
In 4.3 the Configurator should not be in charge of calculating values by itself.
The firmware must tell to the configurator the values.

Also it creates the following bug. Steps:
1. start from defaults - sliders ON
2. CLI: set gyro_lpf2_static_hz = 666
3. Open UI - the value is shown as 500 instead of 666 because it has been calculated in the configurator.

I believe in this PR i see other places where Configurator calculates the filter values based on sliders.

Reply:
@limon this code deals with enabling slider switches making sure they align with the chosen slider value.